pkg/commands/set_cluster_test.go

```diff
@@ -70,3 +70,26 @@ func TestSetClusterFields(t *testing.T) {
 
 	}
 }
+
+func TestSetCiliumFields(t *testing.T) {
+
+	fields := []string{
+		"cluster.spec.networking.cilium.ipam=eni",
+		"cluster.spec.networking.cilium.enableNodePort=true",
+		"cluster.spec.networking.cilium.disableMasquerade=true",
+		"cluster.spec.kubeProxy.enabled=false",
+	}
+
+	cluster := &kops.Cluster{
+		Spec: kops.ClusterSpec{
+			Kubelet: &kops.KubeletConfigSpec{},
+		},
+	}
+
+	var igs []*kops.InstanceGroup
+
+	err := SetClusterFields(fields, cluster, igs)
+	if err != nil {
+		t.Errorf("unexpected error from setClusterFields %v: %v", cluster, err)
+	}
```

Comment:
can you assert that the fields are correctly set in the `cluster` object?

```
if cluster.Spec.Networking.Cilium.Ipam != "eni" {
	t.Errorf("...")
}
```
etc

Reply:
Yep. Just since there are pointers involved, I need to properly demarshal the spec to yaml, I guess